app/page.tsx

```diff
@@ -1,25 +1,25 @@
-import HomeHero from "@/sections/Heros/HomeHero";
-import QuestionCardSection from "@/sections/QuestionCardSection";
-import WhatDoIDoSection from "@/sections/WhatDoIDoSection";
-import HelpRightNowSection from "@/sections/HelpRightNowSection";
-import { Stack } from "@mui/material";
-import PurpleHeading from "@/sections/PurpleHeading";
+import VideoHero from '@/sections/Heros/VideoHero';
+import QuestionCardSection from '@/sections/QuestionCardSection';
+import WhatDoIDoSection from '@/sections/WhatDoIDoSection';
+import HelpRightNowSection from '@/sections/HelpRightNowSection';
+import { Stack } from '@mui/material';
+import PurpleHeading from '@/sections/PurpleHeading';
 
 export default function Home() {
-  return (
-    <>
-      <HomeHero />
```

Comment:
@charcarr Up to you if you think we should delete the old HomeHero from the source code? I think maybe keep it in case they change the hero ? But you're call